include/ofxPixels.h

```diff
@@ -78,7 +78,7 @@ typedef struct OfxRGBColourF {
 
 /** @brief Defines a double precision floating point component RGB pixel */
 typedef struct OfxRGBColourD {
-  double r, g, b, a;
+  double r, g, b;
```

Comment:
This seems like a breaking change, since it changes the size of the struct.  I know it's supposed to be ignored, and I don't think any API function currently uses it directly (though the Support lib does), but I don't think we can just change this.